telemetry/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:18.17.0
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install dependencies
+COPY package.json ./
```

Comment:
Depending on your workflow, I would prefer mounting the entire telemetry folder as a volume into `/usr/src/app` instead of copying files over on container build which would require rebuilding for every change.

Reply:
Good idea, will do. Thanks :)